src/python/pants/backend/project_info/tasks/idea_plugin_gen.py

```diff
@@ -77,6 +80,14 @@ def register_options(cls, register):
                   'jdk name for the --java-language-level is used')
     register('--java-language-level', type=int, default=8,
              help='Sets the java language and jdk used to compile the project\'s java sources.')
+    register('--callback-export-style', type=cls.CallbackExportStyle,
+             default=cls.CallbackExportStyle.raw_specs,
+             help='What IntelliJ would use to call `./pants export` with.'
```

Comment:
It would be good to name and explain this more from an end-user perspective: ie: one of these allows new targets to be loaded as BUILD files are edited that match the `spec`, while the other doesn't, but does apply CLI filters like `tag`.

I think the word `callback` is probably also extraneous in the name... just `--export-style` maybe.